src/interaction/VerletListVSphereInteractionTemplate.hpp

```diff
@@ -127,9 +127,7 @@ namespace espressopp {
         if(potential._computeForce(force, fsi, fsj, p1, p2)) {
           p1.force() += force;
           p2.force() -= force;
-          p1.fradius() = fsi;
```

Comment:
This property is used in  ```integrator/VelocityVerletOnRadius.cpp```;
Could you explain a bit why do you remove those lines (for record purpose) ?